src/Google.Api.Gax/Api/Gax/GaxPreconditions.cs

```diff
@@ -6,6 +6,7 @@
  */
 
 using System;
+using Google.Protobuf;
```

Comment:
Humbug. That was due to a *different* change that was unwanted. After squashing, we should end up not having any changes in here. Sorry!